### PR TITLE
some descriptions improvements

### PR DESCRIPTION
**Pull request by jimkont**
DBPedia -> DBpedia
RDFUnit description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -610,7 +610,7 @@ ARCHIVE - inactive projects or old academic projects that may lack soruce code
 
 ## DevOps
 - [r43ples](https://github.com/plt-tud/r43ples) - Revision Management for the Semantic Web
-- [RDFUnit](https://github.com/AKSW/RDFUnit) - RDFUnit is implemented on top of the Test-Driven Data Validation Ontology and designed to read and produce RDF that complies to that ontology only. 
+- [RDFUnit](https://github.com/AKSW/RDFUnit) - RDFUnit is unit testing framework that can be used either as a JVM (Java) RDF validation library or integrated in a CI/CD environment. RDFUnit checks if the given RDF data conforms to a set of OWL/RDFS ontologies or against SHACL constraints.
```
Comment (maximveksler) [inline README.md]:
```suggestion
- [RDFUnit](https://github.com/AKSW/RDFUnit) - SHACL Unit Testing Suite for OWL/RDFS ontologies.
```

**Comment by maximveksler** [comment]
@jimkont welcome and thank you for the contribution. 

It appears that for Awesome Lists shorter descriptions and references where viewer can learn more are preferable. I've added a suggestion for the updated description of RDFUnit.

File: README.md (hunk 1 above)
Comment (jimkont) [reply README.md]: a shorter version could be:
`RDF Unit testing and validation framework that supports RDFS, OWL and SHACL`
or just `RDF Unit testing and validation framework`

**Comment by jimkont** [comment]
Hi @maximveksler and thanks for the awesome list you assembled

`SHACL Unit Testing Suite for OWL/RDFS ontologies` is not a very accurate description of RDFUnit, I suggested two alternatives, let me know which one you prefer

**Comment by maximveksler** [comment]
@jimkont I suggest "RDF Unit testing and validation framework" ?

**Comment by maximveksler** [comment]
@jimkont thank you for your contribution!

**Pull request merged**